### ISSUE TITLE
'coerce' ignores 'nullable': True

### ISSUE THREAD

**Issue by andreymal** [pyeve/cerberus#269]
``` python
>>> v = Validator({'foo': {'type': 'integer', 'coerce': int, 'nullable': True}})
>>> v.validate({'foo': None})
False
>>> v.errors
{'foo': ["field 'foo' cannot be coerced: int() argument must be a string, a bytes-like object or a number, not 'NoneType'"]}
```

but expected:

``` python
>>> v.validate({'foo': None})
True
>>> v.errors
{}
>>> v.normalized({'foo': None})
{'foo': None}
```

This behavior can be logical (i can set something like `'coerce': lambda x: int(x) if x is not None else None`), but I do not think it's convenient, because I have to specify `nullable` rule twice: in schema and in callable. I think, `nullable` should be checked first, and if value is None, then `coerce` callable should not be called.

If it's a bug and if architecture allows to fix this, please fix this :)

If it's a feature, please add a note about `nullable` in [value coercion documentation](https://cerberus.readthedocs.io/en/stable/normalization-rules.html#value-coercion) (better with an example).

**Comment by funkyfuture** [comment]
this is similar to #268.

**Comment by belthaZornv** [comment]
Bump, this makes sense..

**Comment by sohaibfarooqi** [comment]
Is this feature released? I am using `version 1.1`, still facing this issue

**Comment by laurentrivard** [comment]
+ 1
